etl/churn.kp/knowledge.md

```diff
@@ -455,6 +464,9 @@ def compute_churn_week(df, week_start, bucket, prefix):
           .filter(df['subsession_start_date'] >= week_start_hyphenated)
           .filter(df['subsession_start_date'] < week_end_excl)
     )
+    
+    # clean some of the aggregate fields
+    current_week = current_week.na.fill(0, ["total_uri_count", "unique_domains_count"])
```

Comment:
The `total_uri_count` field should be summed up for all the records along the same lines as `subsession_length`, I think. I'm not sure how best to capture a representative value per-client for `unique_domains_count`, maybe median?

As it is, the code is using only the newest entry per client, which might be OK for `unique_domains_count`, but may not be desired for `total_uri_count`. If it *is* the desired behaviour, please add a comment explicitly describing the behaviour / reasoning.